pyrk/neutronics.py

```diff
@@ -15,7 +15,9 @@ class Neutronics(object):
     """
 
     def __init__(self, iso="u235", e="thermal", n_precursors=6, n_decay=11,
-                 timer=Timer(), rho_ext=None, feedback=False):
+                 n_reflector=0, Lambda_ref=0, ref_rho=[], ref_lambda=[],
```

Comment:
The multi-point model isn't only for reflectors. This naming scheme ("ref", "reflector") muddies the distinction between model and application. Let's think about how best to think of this from a generic modeling standpoint. Perhaps this is an appropriate place for calling an alternative model, rather than combing two different models in a single class.